packages/wonder-stuff-core/src/__tests__/keys.test.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import {keys} from "../keys.js";
+
+describe("#keys", () => {
+    it("should call Object.keys with the given object", () => {
+        // Arrange
+        const keysSpy = jest.spyOn(Object, "keys");
+        const obj = {
+            a: 1,
+            b: "2",
+            c: [3, 4],
+        };
+
+        // Act
+        keys(obj);
+
+        // Assert
+        expect(keysSpy).toHaveBeenCalledWith(obj);
+    });
+
+    it("should return the result of Object.keys", () => {
+        // Arrange
+        const obj = {
+            a: 1,
+            b: "2",
+            c: [3, 4],
+        };
+        jest.spyOn(Object, "keys").mockReturnValueOnce([1, 2, 3]);
```

Comment:
nit: having the mock return `[1,2,3]` instead of `["a", "b", "c"]` makes for a confusing test.

Reply:
Hmm, I actually think using realistic values tends to make it less obvious what's mocked vs. not. In the interest of brevity and clarity of intent, I might prefer something even more obviously not-real:

```js
jest.spyOn(Object, "keys").mockReturnValueOnce("the return value of Object.keys");
expect(keys({})).toBe("the return value of Object.keys");
```